.gitignore

```diff
@@ -7,3 +7,7 @@ data
 .mutagen.yml
 vendor/
 .testchains
+
+x/ibc-rate-limit/contracts/rate-limiter/target/
```

Comment:
what are these paths and what generates [files in] them?

Reply:
contract code was stored and built right in a module,  this is a rudiment